sandbox/reference/app/lib/ui/page/home.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:HyperChat/entity/connection_hub_config.dart';
+import 'package:HyperChat/service/chat_service.dart';
+import 'package:flutter/material.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key key, this.title}) : super(key: key);
+
+  // This widget is the home page of your application. It is stateful, meaning
+  // that it has a State object (defined below) that contains fields that affect
+  // how it looks.
+
+  // This class is the configuration for the state. It holds the values (in this
+  // case the title) provided by the parent (in this case the App widget) and
+  // used by the build method of the State. Fields in a Widget subclass are
+  // always marked "final".
+
+  final String title;
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  ConnectionHubConfig _connectionHubConfig;
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+    final chatService = new ChatService("http://192.168.86.71:8080");
```

Comment:
might be helpful to others to run if can add comments on how you decide on this specific IP:PORT or mention that it could be random.